front/style/pages/_referent.scss

```diff
@@ -37,6 +37,21 @@
                 margin: 0 5px;
             }
         }
+
+        &__search {
+            &__col {
+                flex: 1;
+                margin: 0 100px 0px 0;
```

Comment:
`margin: 0 100px 0 0;` 

or maybe just `margin-right: 100px` ? (if is not an override)